docs/book/installation/upgrading.rst

```diff
@@ -0,0 +1,45 @@
+.. index::
+   single: Upgrading
+
+Upgrading
+=========
+
+Sylius is releasing new versions from time to time. Each release is supported with an UPGRADE file, which is meant to help in the upgrading process,
+especially when the version(major) breaks backwards compatibility.
+
+**Update the Sylius library via Composer by modifying the composer.json file:**
+
+.. code-block:: yaml
+
+    {
+        ...
+
+        "require": {
+            "...": "...",
+
+            "sylius/sylius": "^1.0@beta",
+
+            "...": "...",
+        },
+
+        ...
+    }
+
+**Then run ``composer update`:**
+
+.. code-block:: bash
+
+    $ composer update sylius/sylius
+
+If this results in a dependency error, it may mean that the also other Sylius dependencies have to be upgraded.
```

Comment:
`that the also other` yeah nah
`that other Sylius dependencies also have to be upgraded.` perhaps?